packages/cli/src/commands/spaces/hosts.ts

```diff
@@ -0,0 +1,36 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import heredoc from 'tsheredoc'
+import {displayHostsAsJSON, displayHosts, type Host} from '../../lib/spaces/hosts'
+
+export default class Hosts extends Command {
+  static topic = 'spaces'
+  static hidden = true
+  static description = 'list dedicated hosts for a space'
+  static flags = {
+    space: flags.string({char: 's', description: 'space to get host list from'}),
+    json: flags.boolean({description: 'output in json format'}),
+  }
+
+  static args = {
+    space: Args.string({hidden: true}),
+  }
+
+  public async run(): Promise<void> {
+    const {flags, args} = await this.parse(Hosts)
+    const spaceName = flags.space || args.space
+    if (!spaceName) {
+      ux.error(heredoc(`
+        Error: Missing 1 required arg:
+        space
+        See more help with --help
+      `))
+    }
+
+    const {body: hosts} = await this.heroku.get<Host[]>(`/spaces/${spaceName}/hosts`)
```

Comment:
The command is failing. I believe it has to do with a missing variant header.

```suggestion
    const {body: hosts} = await this.heroku.get<Host[]>(
      `/spaces/${spaceName}/hosts`,
      headers: {Accept: 'application/vnd.heroku+json; version=3.dogwood'}
    )
```

<img width="1534" alt="Screenshot 2024-04-29 at 11 05 29 AM" src="https://github.com/heroku/cli/assets/1291288/37ee66e3-f8bd-4f28-8c00-230b03bd87e7">

Reply:
uhh, whoops. thanks! i should update the tests to fix this as well.